test/storage/online_file_source.test.cpp

```diff
@@ -217,6 +217,25 @@ TEST(OnlineFileSource, TEST_REQUIRES_SERVER(RespectPriorExpires)) {
     loop.run();
 }
 
+TEST(OnlineFileSource, TEST_REQUIRES_SERVER(RespectMinimumUpdateInterval)) {
+    util::RunLoop loop;
+    std::unique_ptr<FileSource> fs = std::make_unique<OnlineFileSource>();
+
+    auto start = util::now();
+    Resource resource{Resource::Unknown, "http://127.0.0.1:3000/test"};
+    resource.priorExpires = start + std::chrono::duration_cast<Seconds>(Milliseconds(100));
+    resource.minimumUpdateInterval = Seconds(1);
+
+    std::unique_ptr<AsyncRequest> req = fs->request(resource, [&](Response) {
+        auto wait = util::now() - start;
+        EXPECT_GE(wait, resource.minimumUpdateInterval);
+        EXPECT_LT(wait, resource.minimumUpdateInterval + Milliseconds(10));
```

Comment:
1: [ RUN      ] OnlineFileSource.RespectMinimumUpdateInterval
1: ../test/storage/online_file_source.test.cpp:232: Failure
1: Expected: (wait) < (resource.minimumUpdateInterval + Milliseconds(10)), actual: 8-byte object <02-00 00-00 00-00 00-00> vs 8-byte object <80-60 33-3C 00-00 00-00>

This one here is flaking.